src/builders/dev-server/index.ts

```diff
@@ -0,0 +1,29 @@
+import { DevServerBuilder } from '@angular-devkit/build-angular';
+import { BuilderContext } from '@angular-devkit/architect';
+import { virtualFs, Path, tags} from '@angular-devkit/core';
+import { SingleSpaBuilderSchema } from '../browser/schema';
+import { Configuration } from 'webpack';
+import * as fs from 'fs';
+import { SingleSpaBrowserBuilder } from '../browser';
+
+
+export class SingleSpaDevServer extends DevServerBuilder {
+    constructor(context: BuilderContext) {
```

Comment:
nit: this constructor is unnecessary, since all we're doing is calling `super`, which is what happens by default if you omit a constructor. So we could remove this entirely

Reply:
sure: I was just following the precedent set in SingleSpaBrowserBuilder for consitency.

Reply:
To keep consistency with what's in the repo already I've left this in for now.

Reply:
fair enough 👍